recipes/galaxy-ml/meta.yaml

```diff
@@ -23,14 +23,14 @@ requirements:
     - cython
     - pip
     - setuptools
-    - numpy>=1.17.1
+    - numpy 1.17.4
   run:
     - python
     - pyfaidx
     - pytabix
     - tabix
     - readme_renderer>=24.0
-    - numpy==1.17.3
+    - numpy 1.17.4
```

Comment:
Ah, 1.17.4 hasn't been put on conda-forge yet, which I suspect is causing the failures on OSX. Do you really need to absolute most recent numpy version?

Reply:
:(. The subversion is not very important, only because the corresponding pypi package has already been pushed to v1.17.5. Why don't those people synchronize their conda and pypi releases??
Anyway, please use the other PR. Thank you @dpryan79

Reply:
There's an issue with numpy on aarch that's holding up the conda packages, I believe. I'll merge in the bot's PR then.